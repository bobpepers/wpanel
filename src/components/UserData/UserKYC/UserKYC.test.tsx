import { shallow } from 'enzyme';
import * as React from 'react';
import { UserKYC, UserKYCProps } from './index';

const defaults: UserKYCProps = {
    user: { phones: [], profile: { country: 'US' } },
    editLabel: jest.fn(),
    alertPush: jest.fn(),
};

describe('UserKYC component', () => {
    const setup = (props: Partial<UserKYCProps> = {}) =>
        shallow(<UserKYC {...{ ...defaults, ...props }} />);

    it('should render', () => {
        const wrapper = setup();
        expect(wrapper).toMatchSnapshot();
        expect(wrapper).toBeDefined();
    });
});
