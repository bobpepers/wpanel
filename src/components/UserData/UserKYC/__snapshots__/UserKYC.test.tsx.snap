// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserKYC component should render 1`] = `
<UserKYCComponent
  alertPush={[Function]}
  classes={
    Object {
      "formControl": "UserKYCComponent-formControl-2",
      "gridColumn": "UserKYCComponent-gridColumn-4",
      "gridRow": "UserKYCComponent-gridRow-3",
      "gridRowWithDivider": "UserKYCComponent-gridRowWithDivider-5",
      "info": "UserKYCComponent-info-7",
      "level": "UserKYCComponent-level-8",
      "paper": "UserKYCComponent-paper-1",
      "title": "UserKYCComponent-title-6",
    }
  }
  editLabel={[Function]}
  user={
    Object {
      "phones": Array [],
      "profile": Object {
        "country": "US",
      },
    }
  }
/>
`;
